### PR TITLE
Updated project doc

### PR DESCRIPTION
**Pull request by jghalam**

### CODE REVIEW

File: docs/lifecycle/lfn_entry/opx.md [hunk 1]
```diff
@@ -1,2 +1,287 @@
...
+
+This document is a template to collect the data LFN thinks are relevant to LFN Project health. Projects will copy this template and instantiate it with their data as a part applying for lifecycle state transitions.
+
+`mkdir -p <relevant lifecycle transition dir>/<project name>/ cp project_data_template.rst !$/project_data.rst`
+
+If the project has instantiated this template before they are welcome to start from that base and update it with any new information. Be sure that the data requested by the primary copy of the Project Data Template has not changed.
+
+These instructions should be removed from the instantiated template.
+
+Here is the file for Tungsten Fabric. It’s a work in progress too:
```
Comment (morganrOL) [inline docs/lifecycle/lfn_entry/opx.md]: shall we keep the ref to TF in opx.md?

File: docs/lifecycle/lfn_entry/opx.md [hunk 2]
```diff
@@ -1,2 +1,287 @@
...
+#### Project release schedule 
+* History of at least two years or age of project 
+    * OPX 2.1.0 – 2017/07/31
+    * OPX 2.2.0 – 2018/02/02
+    * OPX 2.3.0 – 2018/03/28
+    * OPX 2.3.1 – 2018/05/14
+    * OPX 3.0.0 – 2018/09/07
+    * OPX 3.1.0 – 2018/12/20
+* Planned future release schedule
+    * OPX 3.2.0 – TBD
```
Comment (morganrOL) [inline docs/lifecycle/lfn_entry/opx.md:20]: 5 releases in 2018 but none in 2019?

File: docs/lifecycle/lfn_entry/opx.md [hunk 3]
```diff
@@ -1,2 +1,287 @@
...
+            * Merged by committer from same organization as uploader: 0
+            * Merged without substantial code review: 0
+    * Contributor Statistics
+        * Number of contributors: 9
+        * Per-organization
+            * Dell EMC: 8
+            * Inocybe: 1
+
+### Community Current Status
+The project is currently in process of finalizing a draft plan for the increased diversity. The draft plant has been reviewed by the technical commitee. The next step is the review and approval by the OPX board. Here is a link to the draft plan:
```
Comment (morganrOL) [inline docs/lifecycle/lfn_entry/opx.md]: s/plant/plan

File: docs/lifecycle/lfn_entry/opx.md [hunk 4]
```diff
@@ -1,2 +1,287 @@
...
+
+Summary of where candidate project complements functionality already provided by project(s) within LFN
+
+Since OPX runs on an unmodified Linux kernel and enables a lot of networking functionality with the Netlink API, many projects that work with Linux can similarly work with OPX. One such example is FRRouting, which provides a Layer-3 networking control plane – as FRRouting installs routes into the Linux kernel, those routes are also populated in the switch’s networking ASIC (via the NAS Linux adapter) so that the switch can forward traffic accordingly.
+
+Summary of where candidate project overlaps functionality already provided by project(s) within LFN
```
Comment (morganrOL) [inline docs/lifecycle/lfn_entry/opx.md:184]: clear, is there any plan to find synergies?

what link with DANOS if any,  https://www.linuxfoundation.org/networking-orchestration/2018/03/the-linux-foundation-hosts-danos-project-a-unified-network-operating-system/

File: docs/lifecycle/lfn_entry/opx.md (hunk 2 above)
Comment (jghalam) [reply docs/lifecycle/lfn_entry/opx.md:20]: The release of OPX is tied to OS10. We slowed down to 2 releases per year.

**Pull request closed without merging**